ui/src/app/applications/components/utils.tsx

```diff
@@ -119,8 +119,8 @@ export async function deleteApplication(appName: string, appNamespace: string, a
                 }
             }
         },
-        {name: 'argo-icon-warning', color: 'warning'},
-        'yellow',
+        {name: 'argo-icon-warning', color: 'failed'},
+        'red',
```

Comment:
Any specific reason to make this `failed`? Considering this is a warning to the user, we should continue with `warning` itself.

Reply:
The goal is to have the "self" application delete popup in red to make it significantly different from deleting another resource.